benefits/templates/admin/agency-dashboard-index.html

```diff
@@ -0,0 +1,70 @@
+{% extends "admin/base.html" %}
+{% load i18n static %}
+
+{% block extrastyle %}
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet" type="text/css">
```

Comment:
Same here, maybe we just download this font file and serve it locally?

Reply:
I don't think we should do that unless we are actually committed to Roboto, which I don't think we are.

Reply:
@machikoyasuda pointed out that we can actually just remove this line since we don't actually need or use the font in the admin

Reply:
Oh, sorry I meant we're not commited to Robot for the Analog site.

Reply:
👍 removed it in 57b852291a8df3241570d194c2bf13bc1d7547d7